### PR TITLE
44 landpage

### PR DESCRIPTION
**Pull request by MohimaH**
final designs. hopefully

### CODE REVIEW

File: views/add_task.handlebars [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
+    </style>
+</head>
+
+<body>
+    <main>
     <div>
         <h1>Add Task to {{project.name}}</h1>
     </div>
-    <form action='/project_board/{{project.id}}/add_task' method="POST">
+    <form class="project_boardgit"
```
Comment (leighrchristie) [inline views/add_task.handlebars:16]: should there be a git at the end of "project_boardgit"? I can't see a class called that

File: views/landing_page.handlebars [hunk 2]
```diff
@@ -1,21 +1,114 @@
...
+            <label>Already have a Project Board?</label>
+            <br>
+            <button>View All Project Boards</button>
+            <br>
+          </form>
+            <br>
+    
+
+        
     </aricle>
```
Comment (leighrchristie) [inline views/landing_page.handlebars:112]: <article> missing a t, not sure if it matters too much!

File: views/add_task.handlebars (hunk 1 above)
Comment (MohimaH) [reply views/add_task.handlebars:16]: oh I don't know. I didn't do that.

File: views/landing_page.handlebars (hunk 2 above)
Comment (MohimaH) [reply views/landing_page.handlebars:112]: oh thank you. I don't think it does but I will fix that anyway

**Pull request merged**